src/features/listing-builder/components/Form/TitleAndType.tsx

```
@@ -171,13 +181,78 @@ function Type() {
+    name: 'compensationType',
+    control: form.control,
+  });
   const currentHackathon = useMemo(() => {
     return hackathons?.find((h) => h.id === hackathonId);
   }, [hackathonId, hackathons]);
+
+  const handleTypeChange = (newType: string) => {
+    if (newType !== 'bounty' && newType !== 'project') {
+      form.setValue('type', 'hackathon');
+      const hackathon = hackathons?.find((s) => s.slug === newType);
+      if (!!hackathon) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)